src/redux/categories/CatContainer.jsx

```diff
@@ -0,0 +1,26 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { checkStatus } from './catSlice';
+
+const CatContainer = () => {
+  const status = useSelector((state) => state.categories.categories);
+  const dispatch = useDispatch();
+
+  // Dispatch the action immediately (simulate an event or condition)
+  dispatch(checkStatus('Under construction'));
+
+  return (
+    <div className="CategoriesComponent">
+      {status === 'Under construction' ? (
+        <h2 className="notFound"> Status: Under construction</h2>
+      ) : (
+        <ul>
+          {status.map((category) => (
+            <li key={category.id}>{category.name}</li>
+          ))}
+        </ul>
+      )}
+    </div>
+  );
+};
+
+export default CatContainer;
```

Comment:
- [ ] **[OPTIONAL]** Good job with your project, just have a small comment consider moving all your jsx files to the components folder for a better organization. Try to use the redux folder only for redux code.